.ci/install_virtiofsd.sh

```diff
@@ -21,8 +23,10 @@ main() {
 
 	pushd $katacontainers_repo_dir
 	sudo -E PATH=$PATH bash ${buildscript} --build=virtiofsd
-	sudo tar -xvJpf build/kata-static-virtiofsd.tar.xz -C /
-	sudo ln -sf /opt/kata/libexec/virtiofsd /usr/libexec/virtiofsd
+	sudo tar -xvJpf build/kata-static-virtiofsd.tar.xz -C "${DESTDIR}"
+	[ -d "${DESTDIR}/usr/libexec/virtiofsd" ] && \
```

Comment:
Shouldn't it be "-x" ?

Reply:
The check itself doesn't quite make sense since `${DESTDIR}/usr/libexec/virtiofsd` isn't created yet at this point. I guess you mean `${DESTDIR}/opt/kata/libexec/virtiofsd`.

BTW the intent behind this check is to fail early if the `virtiofsd` binary isn't found, right ? If so, I'd rather make this explicit with something like `check || return 1`

Reply:
ops, made a mistake... I wanted to check the existence of `/usr/libexec` directory. It might happen that directory doesn't exist on `$DESTDIR`, that's the case on openshift ci build environment, so we won't attempt to create the link. I will fix that.

Reply:
IIRC on Kata CI the path to virtiofsd in `configuration.toml` is `/usr/libexec/virtiofsd` so the need of the link. Whereas on openshift-ci, everything kata-related should be installed under `/opt/kata` and `configuration.toml` already have the path to virtiofsd correctly, thus it doesn't need `virtiofsd` under `/usr/libexec`. More than that, the kata installation script on openshift CI cannot copy files over `/usr` as it is a read-only directory, and so having that link will introduce more problem.

Reply:
done